static/lang/re-en.json

```
@@ -1165,7 +1165,7 @@
                     "Wolf": "Wolf"
                 },
                 "Dragon": {
-                    "NormalRageDamage": "Deal normal rage damage",
+                    "ToggleLabel": "Dragon Instinct Rage",
```

```suggestion
                    "ToggleLabel": "Draconic Rage",
```
After the class feature